leapp/cli/__init__.py

```diff
@@ -3,6 +3,7 @@
 
 from leapp.utils.clicmd import command, command_opt
 import leapp.cli.upgrade
+import leapp.cli.report
```

Comment:
since we talked about having this report optional, we'd have to surround this by try/except

Reply:
Added try except, optional part will be done in different PR...